tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,56 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
+    echo "::group::deploy_nydus_snapshotter"
+	ensure_yq
+
+	local nydus_snapshotter_install_dir="/tmp/nydus-snapshotter"
+	if [ -d "${nydus_snapshotter_install_dir}" ]; then
+		rm -rf "${nydus_snapshotter_install_dir}"
+	fi
+	mkdir -p "${nydus_snapshotter_install_dir}"
+	nydus_snapshotter_version=$(get_from_kata_deps "externals.nydus-snapshotter.version")
+	git clone -b "${nydus_snapshotter_version}" https://github.com/containerd/nydus-snapshotter "${nydus_snapshotter_install_dir}"
```

Comment:
Please read the repo url from `versions.yaml` too:

https://github.com/kata-containers/kata-containers/pull/8585/commits/4cc0889f0e592e2063f5657a096c4eae4f725e3f#diff-1c4cd801df522f4a92edbfb0fea95364ed074a391ea47c284ddc078f512f7b6aR293

Reply:
Fixed.